orgbee/src/app/pages/freedom-wall/freedom-wall.component.ts

```diff
@@ -1,10 +1,49 @@
 import { Component } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { PostDialogComponent } from '../../post-dialog/post-dialog.component';
 
 @Component({
   selector: 'app-freedom-wall',
   templateUrl: './freedom-wall.component.html',
   styleUrl: './freedom-wall.component.css'
 })
 export class FreedomWallComponent {
+  posts: any[] = [];
+  newPostText: string = '';
+  showPostCreation: boolean = false;
+  showModal: boolean = false;
```

Comment:
Dont use any type. I think you can write it like this.


```suggestion
   posts: Post[] = [];
  newPostText: string = '';
  showPostCreation: boolean = false;
  showModal: boolean = false;
```

reference: https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g26feb3ebb2b_0_0